packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -1591,6 +1592,23 @@ export async function replayTest<
 	await runTestForSeed(model, options, seed, saveInfo);
 }
 
+export function generateTestSeeds(testCount: number, stressMode: StressMode): number[] {
```

Comment:
this function still doesn't really match the implementation I had in mind. I think we fundamentally want 3 sorts of flavors for stress tests:

1. Short - runnable in normal CI pipeline, deterministic seeds
2. Medium - can take longer and have dedicated CI pipeline, determinstic seeds
3. Long - different executions will run different seeds, so repeating the same test command repeatedly will give different results. Repeating a single test seed should still be deterministic though. Usable in e.g. a pipeline on a schedule or in a cron job that's "searching for bugs"

This is mismatched from the current implementation in a few ways:
1. For the deterministic variants, you don't really even need to bother with the whole "pick a random initial seed" bit. You could just use the seeds `0 ... 9` and `0 ... 99` (or whatever constants you pick, potentially favoring the constant passed to `describeFuzz` instead or something like that)
2. For non-deterministic variants, you don't want to mod out by `testCount`. This reduces the potential space of seeds you will run. If you start at seed 12512334, you really do want to just run seeds 12512334 through 12513334 or whatever, not seeds 0 through 1000.